includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -76,10 +80,10 @@ public function __construct() {
 			add_filter( 'give_get_settings_gateways', [ $this, 'register_settings' ] );
 			add_filter( 'give_get_sections_advanced', [ $this, 'register_advanced_sections' ] );
 			add_filter( 'give_get_settings_advanced', [ $this, 'register_advanced_settings' ], 10, 1 );
-			add_action( 'give_admin_field_stripe_account_manager', [ $this, 'stripe_account_manager_field' ], 10, 2 );
+			Hooks::addAction( 'give_admin_field_stripe_account_manager', AccountManagerSettingField::class, 'handle' );
+			Hooks::addAction( 'give_admin_field_stripe_credit_card_format', CreditCardSettingField::class, 'handle', 10, 2 );
```

Comment:
Do this in a Service Provider in `src`. We want to avoid the legacy code referring to new code (though new code can refer to legacy). The reason is we want the dependency to be unidirectional, which allows us to deprecate/remove legacy code without risk of breaking new code.

Reply:
@JasonTheAdams Good catch: https://github.com/impress-org/givewp/compare/dad408679f32...5528061fc6ca